src/EnergyPlus/HVACUnitaryBypassVAV.cc

```diff
@@ -3603,23 +3613,24 @@ namespace HVACUnitaryBypassVAV {
         }
 
         //   Account for floating condition where cooling/heating is required to avoid overshooting setpoint
-        if (CBVAV(CBVAVNumber).HeatCoolMode == 0 && CBVAV(CBVAVNumber).OpMode == DataHVACGlobals::ContFanCycCoil) {
-            if (OutAirTemp > TSupplyToCoolSetPtMin) {
-                CalcSetPointTempTarget = TSupplyToCoolSetPtMin;
-            } else if (OutAirTemp < TSupplyToHeatSetPtMax) {
-                CalcSetPointTempTarget = TSupplyToHeatSetPtMax;
-            } else {
-                CalcSetPointTempTarget = OutAirTemp;
+        if (CBVAV(CBVAVNumber).HeatCoolMode == 0) {
+            if (CBVAV(CBVAVNumber).OpMode == DataHVACGlobals::ContFanCycCoil) {
```

Comment:
This code hasn't changed, I just moved the additional conditional inside so it doesn't have to be evaluated every time through unless HeatCoolMode = 0. Speed up change.